client/landing/jetpack-cloud/components/missing-credentials/index.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	settingsLink: string;
+}
+
+const JetpackCloudMissingCredentialsWarning: FunctionComponent< Props > = ( { settingsLink } ) => {
+	const translate = useTranslate();
+
+	return (
+		<div className="missing-credentials">
+			<div className="missing-credentials__header">
+				<img
+					src="/calypso/images/jetpack/jetpack-connection-bad.svg"
+					alt="jetpack cloud disconnect warning"
+				/>
+			</div>
+			<h4>{ translate( 'Restores not available. Missing server credentials.' ) }</h4>
+			<p>
+				{ translate(
+					'Enter your server credentials to enable one-click restores from your backups.'
```

Comment:
This is not exactly true. 
This component is making the assumption that the user has backup. Should we not take the scan only case into account as well?

Reply:
with the updated selector this is now only shown when rewind credentials are missing. If we want to retrofit the component later for scan we can cross that bridge when we get there

Reply:
👍